### DISCUSSION TITLE
stpyvista-geo

### DISCUSSION THREAD

**Discussion by tkoyama010** [bjlittle/geovista#774] [Show and Tell]
I am now following the @edsaac 's work and found this project. It is fascinating :exploding_head: 

<h3 align="center">
  Show <a href="https://github.com/bjlittle/geovista">GeoVista</a> visualizations in Streamlit using <a href="https://github.com/edsaac/stpyvista">stpyvista</a>.
</h3>

<p align="center">
<a href="https://github.com/edsaac/stpyvista-geo">
<img src="https://github.com/edsaac/stpyvista-geo/blob/main/assets/geovista.gif?raw=true" alt="Screen recording of stpyvista and geovista">
</a>
</p>

**Comment by bjlittle** [comment]
@tkoyama010 It's fantastic, right? 🤩

@edsaac and I touched base a while back, and I've been following their integration of geovista with stpyvista.

I totally ♥️ it!

They're doing amazing work to extend streamlit integration with pyvista. A big win for the community!

**Reply by tkoyama010** [reply]
I would like to share Dr. @giswqs's video in this thread. It is very inspirational to develop a geo web app using Streamlit.

📺 [Build Interactive Geospatial Web Apps with Leafmap and Streamlit
](https://youtu.be/hn2WqRX75DI?si=lGTqUqfJTpxivuiR)

**Comment by edsaac** [comment]
thanks! I did not realize someone was checking that demo! It was crashing; I just put it back online again:

[![Geovista & Streamlit demo](https://img.shields.io/static/v1?label=%20&message=Open%20in%20Community%20Cloud&color=pink&logo=streamlit)](https://geovista.streamlit.app/)

**Reply by kuchaale** [reply]
Not anymore accessible.

**Reply by edsaac** [reply]
Just click on the "Yes, get this app back up!" button. When there is no traffic to a Streamlit app hosted in their cloud service, they put it to sleep.

**Comment by tkoyama010** [comment]
~@allcontributors please add @edsaac for plugin~
